aman/src/pages/Forgotpsswrd.jsx

```diff
@@ -0,0 +1,82 @@
+import { z } from "zod";
+import { useState } from "react";
+import Input from "../components/Input";
+const Forgotpsswrd = ({ handleClick }) => {
+  const [formData, setFormData] = useState({ email: "" });
+  const [error, setError] = useState({});
+  const schema = z.object({
+    email: z.string().email("Must be a valid email address"),
+  });
+
+  const handleChange = (e) => {
+    setFormData({ ...formData, [e.target.name]: e.target.value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const result = schema.safeParse({
+      ...formData,
+    });
+    if (!result.success) {
+      const formattedErrors = result.error.format();
+      setError(formattedErrors);
+    } else {
+      setError({});
+    }
+  };
+  return (
+    <div>
+      <div className="flex flex-col items-center justify-center m-4 mt-10">
+        <div className="rounded-lg shadow-lg  w-full max-w-md h-[510px] m-8 p-3  bg-white mb-18">
```

Comment:
Avoid using such properties like h-[510px]. Try to use available tailwind properties. Also, here in this case, instead of adding such height try to add padding and spacing.